lib/radiator_web/graphql/admin/resolvers/session.ex

```diff
@@ -0,0 +1,30 @@
+defmodule RadiatorWeb.GraphQL.Admin.Resolvers.Session do
+  import Radiator.Auth.Register, only: [get_user_by_credentials: 2]
+  import Radiator.Auth.Guardian, only: [api_session_token: 1]
```

Comment:
Was just following a mood to be honest. I think here it works because the function names can stand on their own. Usually it's helpful to see the module for context but here it's not necessary, especially in such a short module.

The reason I touched it at all is because I found the methods hard to read due to their long line lengths and felt this way helps readability.

Reply:
Ah, I see. Differences in philosophies here. I don't have a strong opinion here.